units/Legion/Legion EvoCom/legevodecom.lua

```diff
@@ -13,6 +13,8 @@ unitsTable['legdecomlvl3'].decoyfor = "legcomlvl3"
 unitsTable['legdecomlvl3'].customparams.decoyfor = "legcomlvl3"
 unitsTable['legdecomlvl3'].health = math.ceil(unitsTable['legdecomlvl3'].health*0.5)
 unitsTable['legdecomlvl3'].weapondefs.disintegrator.damage.default = 40
+--unitsTable['legdecomlvl3'].weapondefs.napalmmissile.customparams.stockpilelimit = 1
```

Comment:
Unused weapon but has stockpile configuration so should have limit, or could be deleted entirely

Reply:
Removed unused weapondef